src/main/java/org/gooru/insights/services/CSVFileWriterServiceImpl.java

```diff
@@ -34,14 +34,17 @@ public BaseConnectionService getBaseConnectionService() {
 		return baseConnectionService;
 	}
 	
-	@Override
-	public void generateCSVReport(Set<String> headerKeys, List<Map<String, Object>> rowList, String fileAbsolutePath, String delimiter, Boolean isNewFile) throws FileNotFoundException {
+	public void generateCSVReport(String traceId, Set<String> headerKeys, List<Map<String, Object>> rowList, String fileAbsolutePath, String delimiter, Boolean isNewFile) throws FileNotFoundException {
```

Comment:
@daniel1028 , It will help to pass different delimiter as we need for different type of data export. And I have mentioned default delimiter in constant.